ui/components/ui/tabs/tab/index.scss

```diff
@@ -10,15 +10,21 @@
     color: unset;
     font-weight: 500;
     transition: color 50ms ease, border-color 50ms ease;
+
+    &:not([disabled]) {
+      &:hover {
+        color: var(--color-text-default);
+      }
+    }
   }
 
-  &--single {
-    color: var(--color-text-default);
-    border-bottom: none;
+  &--disabled {
+    color: var(--color-text-muted);
   }
 
-  &:hover {
+  &--single {
     color: var(--color-text-default);
+    border-bottom: none;
   }
```

Comment:
Unfortunate formatting of changes here, but we’re retaining the same `&--single` styles. The hover styles are now scoped under the `button` specifically to ensure they aren't applied to a disabled button.

```scss
  &--disabled {
    color: var(--color-text-muted);
  }

  &--single {
    color: var(--color-text-default);
    border-bottom: none;
  }
```